react-ui/src/pages/entry/EntryModal.jsx

```diff
@@ -1,30 +1,29 @@
 /**
  * Full page entry
+ * ??? Get rid of Refs for click() events, just open link
  */
 
 import Box from '@material-ui/core/Box';
+import Dialog from '@material-ui/core/Dialog';
 import { makeStyles } from '@material-ui/core/styles';
-import FlagTwoToneIcon from '@material-ui/icons/FlagTwoTone';
-import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
 import NavigateBeforeIcon from '@material-ui/icons/NavigateBefore';
 import NavigateNextIcon from '@material-ui/icons/NavigateNext';
-import RedditIcon from '@material-ui/icons/Reddit';
 import clsx from 'clsx';
 import throttle from 'lodash/throttle';
 import {
   useEffect, useMemo, useRef, useState,
 } from 'react';
-import { useParams, useLocation, useNavigate } from 'react-router-dom';
+import { useParams, useLocation, useNavigate, useOutletContext } from 'react-router-dom';
```

Comment:
eslint:
`Expected a line break after this opening brace`
`Expected a line break before this closing brace`